### ISSUE TITLE
Update CI cache to avoid downloading artefacts

### ISSUE THREAD

**Issue by smarter** [lampepfl/dotty#2290]
http://dotty-ci.epfl.ch/lampepfl/dotty/1899/3 :
```
[info] downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-asm/5.1.0-scala-2/scala-asm-5.1.0-scala-2.jar ...
[info] 	[SUCCESSFUL ] org.scala-lang.modules#scala-asm;5.1.0-scala-2!scala-asm.jar(bundle) (190ms)
[info] downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.11/1.0.1/scala-xml_2.11-1.0.1.jar ...
[info] [SUCCESSFUL ] org.scala-lang.modules#scala-xml_2.11;1.0.1!scala-xml_2.11.jar(bundle) (270ms)
```

**Comment by smarter** [comment]
This is more important now that we upgraded to 2.11.11

**Comment by DarkDimius** [comment]
I think that updating CI cache every month is too much of a hussle.
Could we achieve the same result by hosting either a maven mirror or squid reverse cache on the same machine?

**Comment by felixmulder** [comment]
We could add a hook that publishes the dotty-docker image to latest and some timestamp. That would solve it.

**Comment by adriaanm** [comment]
Artifactory is relatively simple to set up. Here's a recent dump of our config: https://gist.github.com/SethTisue/4ef883260582d06f29ea3fb065f6fde9

**Comment by smarter** [comment]
@felixmulder @OlivierBlanvillain CI cache needs to be updated again after the switch to 2.12, not sure who's responsibility this is now :).

**Comment by OlivierBlanvillain** [comment]
Fixed in #3035. For future reference instructions to rebuild the docker image are in https://github.com/lampepfl/dotty-drone.
